### ISSUE TITLE
How do I change the batch_size?

### ISSUE THREAD

**Issue by DongwhanLee** [hq-jiang/instance-segmentation-with-discriminative-loss-tensorflow#18]
Hi, I am Dongwhan Lee from Korea.

I am impressed by your work and trying to follow yours.
However, I encountered the error like below:

![image](https://user-images.githubusercontent.com/70641179/113986186-ebdbf580-9887-11eb-9902-5c318e4c37b6.png)

That error message appears after I changed batch_size=1 to batch_size=16 in "training.py".
I looked into "datagenerator.py" too but failed to how to fix the error.

Is there any way to change the batch_size? If so, could you let me know?
Plus, I also wonder where that 1 in (1, 512, 512, 3) comes from.

Thank you in advance.

**Comment by parthdave1210** [comment]
Hi, This is Parth Dave.

I am trying to run this program on my machine. Can you please give me the Requirements to run the code?

**Comment by jinghanxu1999** [comment]
I also have this problem, have you solved it? And can you tell me how did you solved it ?think you
